api/services/webhookService.js

```diff
@@ -0,0 +1,27 @@
+const send = async (payload) => {
+  const https = require('https')
+
+  const data = JSON.stringify(payload)
+
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Content-Length': Buffer.byteLength(data),
+      'Authorization': `Bearer ${process.env.GHBS_WEBHOOK_SECRET}`
```

Comment:
Needs to be Authorization: Token X to match the rails side.
https://github.com/DFE-Digital/buy-for-your-school/pull/1045/files#diff-7c744220367125260d617510d3d57e6bccb5e04286a627dd92dc268e27bf8323R14